packages/utop/utop.2.2.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "jeremie@dimino.org"
+authors: ["Jérémie Dimino"]
+license: "BSD3"
+homepage: "https://github.com/diml/utop"
+bug-reports: "https://github.com/diml/utop/issues"
+dev-repo: "https://github.com/diml/utop.git"
+build: [
+  ["jbuilder" "subst"] {pinned}
```

Comment:
Could you add `"-p" name` here? See https://github.com/ocaml/opam-repository/pull/12279